### DISCUSSION TITLE
Support for Calibre

### DISCUSSION THREAD

**Discussion by M-Davies** [Luctia/ezarr#38] [Ideas]
https://hub.docker.com/r/linuxserver/calibre-web
https://hub.docker.com/r/linuxserver/calibre

Seeing as this stack seems to be missing a dedicated Ebook reader (minus audiobookshelf but that's more of a audiobook/podcast soloution), having one would be really useful in cominbation with readarr

**Comment by M-Davies** [comment]
@Luctia Would love to get your thoughts on this, happy to look into making a PR if you need a hand

**Reply by Luctia** [reply]
I like the idea! If it's not too much work, a PR would be nice. I'll create an issue for it 👍

**Comment by Luctia** [comment]
Closing, as an issue has been created for it.
